ldk/javascript/package.json

```diff
@@ -18,6 +18,7 @@
   ],
   "scripts": {
     "build": "rm -rf dist && tsc",
+    "prepush": "npm run build && npm run test && npm run lint",
```

Comment:
*high five*

Reply:
hehe, 1 step at a time.

Reply:
A thought also occurred to me - what about including building the `examples` dir as well to help prevent those projects from going stale?

Reply:
Hmm, I think it is a good idea to run them, but I'm wondering where would be best to do it. Perhaps it is better fits to do it in github actions via hooks?

Reply:
Maybe we can create a new user story for this.